index.html

```diff
@@ -63,7 +63,7 @@
     <nav class="navbar navbar-default navbar-custom">
         <div class="container">
             <div class="navbar-header">
-                <a href="/~wics/"><img style=".img-fluid. max-width: 100%; height: 50px;" src="/~wics/assets/img/favicon/favicon-310.png"/></a>
+                <a href="/~wics/"><img class="img-fluid" src="/~wics/assets/img/favicon/favicon-310.png" alt="Responsive image"/></a>
```

Comment:
icon is too big on main page: this img should have style like the red line
<img style=".img-fluid.max-width: 100%;height: 50px;" src="/~wics/assets/img/favicon/favicon-310.png">

Reply:
i can make the icon the same size as the other pages. I only changed it because the ".img-fluid" is the wrong syntax for style specifier